services/QuillLMS/app/workers/evidence_update_scoring_worker.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+class EvidenceUpdateScoringWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: SidekiqQueue::MIGRATION
+
+  attr_reader :concept_results
+
+  def perform(activity_session_id)
+    @concept_results = ConceptResult.where(activity_session_id:)
+
+    return if concept_results.empty?
```

Comment:
Consider adding a check that this is an `Evidence` activity in case this is re-used or there's a bug in how the jobs get queued.